rero_ils/translations/messages.pot

```diff
@@ -5,9 +5,9 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: rero-ils 0.6.0\n"
```

Comment:
Why this file is a part of this commit while the language files are not. Did you forget to run update_catalog?

Reply:
run update catalog done